packages/firestore/src/protos/firestore_proto_api.ts

```diff
@@ -456,6 +456,8 @@ export declare type BeginTransactionRequest =
   firestoreV1ApiClientInterfaces.BeginTransactionRequest;
 export declare type BeginTransactionResponse =
   firestoreV1ApiClientInterfaces.BeginTransactionResponse;
+  export declare type BloomFilter =
```

Comment:
ultranit: I think this line should be un-indented to be consistent with the surrounding lines.